tests/test_interop_run_stats_parser.py

```diff
@@ -26,5 +29,50 @@ def test_lanes_total_clusters(self):
             assert lane._total_clusters_pf is not None and lane._total_clusters_pf != 0
             assert lane._total_clusters_raw is not None and lane._total_clusters_raw != 0
 
+    def test_clusters_same_across_lanes(self):
+        """
+        Verify that 'Reads' and 'Reads Pf' are consistently the same across all reads within the same lane
+        """
+        non_index_reads = [0, 2, 3]
+        runfolder = "tests/resources/200624_A00834_0183_BHMTFYTINY"
+        iop = InteropRunStatsParser(runfolder, non_index_reads)
+        
+
+        data = {
```

Comment:
I think the two first assertions (`Assert all 'Reads' and 'Reads Pf' values are consistent within the lane`) would be more meaningful if we used the test data instead of a mock (where we explicitly define that `Reads` and `Reads Pf` should be the same on all rows). 
It could be achieved by doing something like this: 
```
        interop_lane_summary = interop.summary(iop._run_metrics, 'Lane')
        df = pd.DataFrame(interop_lane_summary)
```

I think the resulting data frame will be compatible with the rest of the test (from line 51).